### ISSUE TITLE
 [Error] When provisioing the k8s cluster, an error occurs when setup.sh running the script. - ImportError: cannot import name 'soft_unicode' from 'markupsafe' 

### ISSUE THREAD

**Issue by jjsair0412** [NVIDIA/deepops#1252]
when i provisioning k8s cluster, I was run setup.sh script file.

But i found some that error and i need help..

os : ubuntu 20.04
deepops version : 22.01
server specification : 4core , memort 8GB
python version : python3

```
Traceback (most recent call last):
  File "/opt/deepops/env/bin/ansible-galaxy", line 62, in <module>
    import ansible.constants as C
  File "/opt/deepops/env/lib/python3.8/site-packages/ansible/constants.py", line 12, in <module>
    from jinja2 import Template
  File "/opt/deepops/env/lib/python3.8/site-packages/jinja2/__init__.py", line 12, in <module>
    from .environment import Environment
  File "/opt/deepops/env/lib/python3.8/site-packages/jinja2/environment.py", line 25, in <module>
    from .defaults import BLOCK_END_STRING
  File "/opt/deepops/env/lib/python3.8/site-packages/jinja2/defaults.py", line 3, in <module>
    from .filters import FILTERS as DEFAULT_FILTERS  # noqa: F401
  File "/opt/deepops/env/lib/python3.8/site-packages/jinja2/filters.py", line 13, in <module>
    from markupsafe import soft_unicode
ImportError: cannot import name 'soft_unicode' from 'markupsafe' (/opt/deepops/env/lib/python3.8/site-packages/markupsafe/__init__.py)
Traceback (most recent call last):
  File "/opt/deepops/env/bin/ansible-galaxy", line 62, in <module>
    import ansible.constants as C
  File "/opt/deepops/env/lib/python3.8/site-packages/ansible/constants.py", line 12, in <module>
    from jinja2 import Template
  File "/opt/deepops/env/lib/python3.8/site-packages/jinja2/__init__.py", line 12, in <module>
    from .environment import Environment
  File "/opt/deepops/env/lib/python3.8/site-packages/jinja2/environment.py", line 25, in <module>
    from .defaults import BLOCK_END_STRING
  File "/opt/deepops/env/lib/python3.8/site-packages/jinja2/defaults.py", line 3, in <module>
    from .filters import FILTERS as DEFAULT_FILTERS  # noqa: F401
  File "/opt/deepops/env/lib/python3.8/site-packages/jinja2/filters.py", line 13, in <module>
    from markupsafe import soft_unicode
ImportError: cannot import name 'soft_unicode' from 'markupsafe' (/opt/deepops/env/lib/python3.8/site-packages/markupsafe/__init__.py)
Traceback (most recent call last):
  File "/opt/deepops/env/bin/ansible", line 62, in <module>
    import ansible.constants as C
  File "/opt/deepops/env/lib/python3.8/site-packages/ansible/constants.py", line 12, in <module>
    from jinja2 import Template
  File "/opt/deepops/env/lib/python3.8/site-packages/jinja2/__init__.py", line 12, in <module>
    from .environment import Environment
  File "/opt/deepops/env/lib/python3.8/site-packages/jinja2/environment.py", line 25, in <module>
    from .defaults import BLOCK_END_STRING
  File "/opt/deepops/env/lib/python3.8/site-packages/jinja2/defaults.py", line 3, in <module>
    from .filters import FILTERS as DEFAULT_FILTERS  # noqa: F401
  File "/opt/deepops/env/lib/python3.8/site-packages/jinja2/filters.py", line 13, in <module>
    from markupsafe import soft_unicode
ImportError: cannot import name 'soft_unicode' from 'markupsafe' (/opt/deepops/env/lib/python3.8/site-packages/markupsafe/__init__.py)
```

**Comment by jungyh0218** [comment]
As far as I know, the issue is fixed in version 22.04. 
https://github.com/NVIDIA/deepops/pull/1114

**Comment by jjsair0412** [comment]
I will test that version and share the results.

Thank you for your help.
